test/test.js

```diff
@@ -0,0 +1,83 @@
+var Queue = require('../lib/Queue.js');
+var should = require('chai').should();
+
+describe('Queue', function() {
+  describe('Class methods', function() {
+    var myQueue;
+    beforeEach(function() {
+      myQueue = new Queue();
+    });
+    it('should have a size() function', function() {
+      myQueue.size.should.be.a('function');
+    });
+    it('should have a enqueue() function', function() {
+      myQueue.enqueue.should.be.a('function');
+    });
+    it('should have a dequeue() function', function() {
+      myQueue.dequeue.should.be.a('function');
+    });
+    it('should have a peek() function', function() {
+      myQueue.peek.should.be.a('function');
+    });
+  });
+
+  describe('Queue behavior', function() {
```

Comment:
Currently with this title the output is:

`1) Queue Queue behavior A new queue should start with a size of 0:`

consider removing "Queue" from this describe block so the output is instead:

`1) Queue behavior A new queue should start with a size of 0:`